views/topic/edit-topic.hbs

```diff
@@ -44,10 +44,9 @@
 
                 <div class="form-group">
                     <label>{{$t "topic._topic.label.content" }}</label>
-
                     <!-- React Magic -->
-                    <div id="content-blocks" data-value="{{lesson.contents}}" data-parent-id="{{lesson._id}}" data-school-id="{{schoolId}}" data-parent-type="lessons" data-etherpadbaseurl="{{etherpadBaseUrl}}"
-                        data-iscoursegroup="{{courseGroupId}}" data-nexboardenabled="{{#ifEnv "FEATURE_NEXBOARD_ENABLED" "true"}}true{{/ifEnv}}"></div>
+                    <div id="content-blocks" data-testid="nexboardButton" data-value="{{lesson.contents}}" data-parent-id="{{lesson._id}}" data-school-id="{{schoolId}}" data-parent-type="lessons" data-etherpadbaseurl="{{etherpadBaseUrl}}"
+                        data-iscoursegroup="{{courseGroupId}}" {{#unlessEnv "FEATURE_NEXBOARD_ENABLED" "true"}} data-neboardenabled="false" {{/unlessEnv}}></div></div>
```

Comment:
I have a hard time understanding the logic here. Please correct me if I'm wrong, but to me it looks like that: unless the `FEATURE_NEXBOARD_ENABLED` will be set to `true`, the `data-neboardenabled="false"` (BTW you've got a type here, I guess it should be data-ne**x**boardenabled) will be added which basically disables the Nexboard button so in the end when `FEATURE_NEXBOARD_ENABLED` will be set to `false`, the `data-neboardenabled="false"` will not be added and thanks to this the button also will not be added as I guess it's only added when the `data-neboardenabled` is set to `true` (which also means it has to be present right)?

So does it mean the button will be always disabled, in any case so the clause doesn't really do anything?

Reply:
The {{#unlessEnv "FEATURE_NEXBOARD_ENABLED" "true"}} ... {{/unlessEnv}} block dynamically adds data-**nexboardenabled="false"** to the <div> element only if the **FEATURE_NEXBOARD_ENABLED** is not set to true.
This means, if **FEATURE_NEXBOARD_ENABLED** is false or not defined in the environment, data-**nexboardenabled="false"** is added to the <div>. **If FEATURE_NEXBOARD_ENABLED** is true, this attribute won’t be added.

The JavaScript logic that renders the NeXboard button should check the data-nexboardenabled attribute. If it’s "false" or not present, the button won’t be rendered (or will be hidden), effectively controlling the button's visibility based on the FEATURE_NEXBOARD_ENABLED setting.

Reply:
> If FEATURE_NEXBOARD_ENABLED is true, this attribute won’t be added.
Exactly. That's why the button will not be rendered. So it'll never be rendered, in any case.